client/layout/guided-tours/tours/jetpack-checklist-tour/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import { ButtonRow, makeTour, Next, Quit, Step, Tour } from 'layout/guided-tours/config-elements';
+
+export const JetpackChecklistTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			arrow="bottom-left"
+			name="init"
+			placement="above"
+			style={ {
+				marginTop: '-20px',
+			} }
+			target={ '.checklist__header-main' }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
```

Comment:
Drive-by nit:

I'm rather partial to writing `Fragment`s as `<> … </>`, saves the need to add another import or reference `<React.Fragment> … </React.Fragment>`.

https://reactjs.org/docs/fragments.html#short-syntax

Reply:
I prefer `<>` myself as well but I also prefer consistency over anything else, so I've stuck to Fragment in Calypso until now. Looks like  `<>` has crept up in numbers tho so ok to switch. :-)

Reply:
I think `<Fragment />` has been lingering from times when our toolchain (babel upgrade blocked by eslint upgrade or whatever) wasn't able to handle `</>` just yet.

Thus +1 on using `</>` in newly written code (and replacing `<Fragment />` in legacy code that we touch.)

Reply:
Done!